terraform/terraform-modules/mwaa-env/requirements.txt

```diff
@@ -1,2 +1,2 @@
 apache-airflow-providers-docker==3.1.0
-apache-airflow-providers-amazon==2.4.3
+apache-airflow-providers-amazon==8.18.0
```

Comment:
Okay to use the compatible release operator here, `~=`, instead of precise `==`?

Reply:
Used the compatible release operator.